Before: b165eb1ae8e07cdd8105c8120e47aa1384f92032a5035cd228cc84f501a6644d
After: 037fdeb0f338acd3b301085ccfe5059d345794c2c8955edf3024af42ba0c2535
Patch: cd499ff9690afaa1615a7d012f8692596c2284d0f5084f145b8ba600e37ecaac
--- .corvid/Gemfile	2012-07-24 18:19:52.226592976 +1000
+++ /dev/null	2012-07-24 17:25:00.561565760 +1000
@@ -1,30 +0,0 @@
-# vim:ft=ruby et ts=2 sw=2:
-
-gem 'corvid'
-
-gem 'rake', group: :rake
-
-group :test do
-  gem 'ci_reporter', require: false
-  gem 'simplecov', '>= 0.6.4', require: false
-end if Dir.exists?('test')
-
-group :test_unit do
-  gem 'minitest'
-  gem 'guard-minitest', '>= 0.5.0', require: false
-  gem 'turn', '>= 0.9.4'
-end if Dir.exists?('test/unit')
-
-group :test_spec do
-  gem 'rspec'
-  gem 'guard-rspec', require: false
-end if Dir.exists?('test/spec')
-
-group :doc do
-  gem 'yard'
-end
-
-group :debug do
-  gem 'debugger', platforms: :mri
-end
-
--- Gemfile	2012-07-24 22:56:12.079632902 +1000
+++ Gemfile	2012-07-24 23:02:34.860672426 +1000
@@ -1,7 +1,7 @@
 source :rubygems
 
-# Load Corvid dependencies
-eval File.read(File.expand_path '../.corvid/Gemfile', __FILE__)
+# Load Gemfile.corvid
+eval File.read(__FILE__ + '.corvid')
 
 # Parser for Markdown documentation
 group :doc do
--- /dev/null	2012-07-24 17:25:00.561565760 +1000
+++ Gemfile.corvid	2012-07-24 23:02:34.860672426 +1000
@@ -0,0 +1,30 @@
+# vim:ft=ruby et ts=2 sw=2:
+
+gem 'corvid'
+
+gem 'rake', group: :rake
+
+group :test do
+  gem 'ci_reporter', require: false
+  gem 'simplecov', '>= 0.6.4', require: false
+end if Dir.exists?('test')
+
+group :test_unit do
+  gem 'minitest'
+  gem 'guard-minitest', '>= 0.5.0', require: false
+  gem 'turn', '>= 0.9.4'
+end if Dir.exists?('test/unit')
+
+group :test_spec do
+  gem 'rspec'
+  gem 'guard-rspec', require: false
+end if Dir.exists?('test/spec')
+
+group :doc do
+  gem 'yard'
+end
+
+group :debug do
+  gem 'debugger', platforms: :mri
+end
+
